spec/models/log_spec.rb

```diff
@@ -24,7 +24,7 @@
 
         context 'when all log entries were created long enough ago that a reminder is needed' do
           before do
-            log.log_entries.find_each do |log_entry|
+            log.log_entries.includes(:log_entry_datum).find_each do |log_entry|
```

Comment:
It's a tiny bit annoying that Prosopite enforces eager loading in spec code, not just application code, but at the same time it's maybe/probably somewhat of a good thing. For one thing, spec performance does matter, too (although, in most cases, the number of records involved will probably make eager loading a relatively negligible cost).